app/main.ts

```diff
@@ -79,9 +90,20 @@ try {
     }
   });
 
-  ipcMain.on("asynchronous-message", (event, arg) => {
-    console.log("copying files");
-    event.reply("asynchronous-reply", "pong");
+  ipcMain.on("check-files-are-created", (event, arg) => {
+    try {
+      setInterval(() => {
+        if (
+          fs.existsSync(userDirectory + "/model.json") &&
```

Comment:
There's a case where we already have those files created. In which case, we should always make sure that the directory is empty. Or delete the files. Then we should be saving them.

Reply:
done